src/coreclr/jit/hwintrinsicxarch.cpp

```diff
@@ -1272,8 +1272,9 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
 
         case NI_Vector128_Divide:
         case NI_Vector256_Divide:
+        case NI_Vector512_Divide:
         case NI_Vector128_op_Division:
-        case NI_Vector256_op_Division:
+        case NI_Vector512_op_Division:
```

Comment:
Intentional that `NI_Vector256_op_Division` was deleted?

Reply:
Thanks for the catch. Fixed.